packages/core/server/index.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable global-require, no-console, import/order */
+const {
+  API_ROOT_URL,
+  API_ORIGIN,
+} = process.env;
```

Comment:
```suggestion
const {
  API_ROOT_URL,
  API_ORIGIN,
  NODE_ENV,
} = process.env;
```

Reply:
Unfortunately (and perhaps interestingly) this isn't something we can do in this case. Webpack will look for exactly `process.env.NODE_ENV` when performing static replacement of that variable with its value and closing of execution contexts. Take a look here: https://webpack.js.org/plugins/environment-plugin/

Reply:
Interesting!